### PR TITLE
Update cwltest to 2.0.20200220223835

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`cwltest`](https://bioconda.github.io/recipes/cwltest/README.html): **1.0.20190906212748** &rarr; **2.0.20200220223835**

[install with bioconda](http://bioconda.github.io/recipes/cwltest/README.html) [Conda](https://anaconda.org/bioconda/cwltest/files)

Info | Link
-----|-----
Recipe | [`recipes/cwltest`](https://github.com//bioconda/bioconda-recipes/tree/bump/cwltest/recipes/cwltest) (click to view/edit other files)
Releases |[https://pypi.org/pypi/cwltest/json](https://pypi.org/pypi/cwltest/json)

***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by mbargull** [comment]
Blocked by https://github.com/conda-forge/schema-salad-feedstock/pull/30.

File: recipes/cwltest/meta.yaml [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
 {% set version = "2.0.20200220223835" %}
 
 package:
-  name: "{{ name|lower }}"
-  version: "{{ version }}"
+  name: cwltest
+  version: {{ version }}
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (marcelm) [inline recipes/cwltest/meta.yaml]: Your change here seems to be in the opposite direction of what I’ve seen in other recipes. Is there a policy somewhere on how PyPI URLs should be written in recipes?

(I agree with your change and find it more readable, but I’m worried others would “correct” this by re-introducing the `{{ name }}` variable.)

Comment (mbargull) [reply recipes/cwltest/meta.yaml]: No, I don't think we have a policy.
The overuse of Jinja templating is just what `conda-skeleton` outputs, hence it is quite common.
Johannes started some work to counteract that bloat in https://github.com/conda/conda-build/pull/3176.
I'm actually not sure where most of the other folks stand on this. At least there was a comment at https://github.com/conda/conda-build/pull/3176#issuecomment-427454913 saying others in conda-forge gradually reduced the amount of template use.
It's not too unlikely someone (Johannes, myself, or maybe Marcelo Trevisani) will work on this some time this year, I guess.

Comment (marcelm) [reply recipes/cwltest/meta.yaml]: Thanks, this was some interesting reading. Then I’m inclined to follow the simpler style myself and I’ll also suggest it when reviewing recipes.

**Comment by mbargull** [comment]
@BiocondaBot please update

**Comment by mbargull** [comment]
@BiocondaBot please fetch artifacts

**Comment by mbargull** [comment]
@BiocondaBot please merge

**Pull request merged**